### DISCUSSION TITLE
Rich Presence Game Icon

### DISCUSSION THREAD

**Discussion by C0an** [discord-jda/JDA#2415] [Questions and Help]
Hi! I noticed that Discord provides game icons that they know about (Overwatch 2, VALORANT, CS:GO) with Rich presence, is there a way to get these game icons with JDA or Discord API in general? 
![image](https://user-images.githubusercontent.com/43456714/223587866-77b3db39-e5f0-49ed-aace-e26384338018.png)

**Comment by MinnDevelopment** [comment]
This is available via [RichPresence#getLargeImage](https://ci.dv8tion.net/job/JDA5/javadoc/net/dv8tion/jda/api/entities/RichPresence.html#getLargeImage()):

```java
RichPresence rp = member.getActivities().get(0).asRichPresence();
RichPresence.Image image = rp.getLargeImage();
System.out.println(image.getUrl());
```
